minor cleanups

thezoggy opened the pull request.

thezoggy committed:
Removed last jui file and correct include to use proper revision.

thezoggy committed:
Clean up sb version line and add the install type (git/win/src).

thezoggy committed:
Highlight the output preview fields that would result in a change from the original filename.

midgetspy commented on SickBeard.py:
```diff
@@ -1,328 +1,341 @@
-#!/usr/bin/env python
```
Endlines? This file needs to be \n so it can be executed directly on linux boxes.

thezoggy replied:
yeah, looks like pyDev did that.. gotta go fix that. sorry

thezoggy committed:
Added `.pydevproject` to `.gitignore`.
Added startup check for Cheetah 2+ (should be helpful for first time users and when users upgrade OSX as they get a fresh copy of python installed).
Cleaned up some comments and made some minor white-space adjustments.

thezoggy committed:
testRename: Changed it so that we highlight the entire row instead of just the newLoc field.

Commit:
Change testRename present season DESC episode DESC on screen

thezoggy committed:
Merge pull request #6 from itofzo/dev-pyDev

Change testRename present season DESC episode DESC on screen

thezoggy committed:
Remove extra `</td>`
